packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { randomId } from '@rentpath/react-ui-utils'
+import Star from './SampleSvgs/Star'
+
+export default class RatingBar extends Component {
+  static propTypes = {
+    theme: PropTypes.object,
+    ratingScale: PropTypes.number,
+    rating: PropTypes.number,
+    total: PropTypes.string,
+    RatingItem: PropTypes.node,
+  }
+  static defaultProps = {
+    theme: {},
+    RatingItem: Star,
+  }
+
+  get ratingItems() {
+    const {
+      ratingScale,
+      rating,
+      RatingItem,
+      ...props
+    } = this.props
+
+    return [...Array(ratingScale).keys()].map(index => (
+      <RatingItem
+        key={randomId('rating')}
+        id={randomId('rating-item')}
+        width={`${(rating - index) * 100}%`}
+        {...props}
+      />
+    ))
+  }
+
+  render() {
+    const {
+      total,
+      theme,
+    } = this.props
+    return (
+      <div className={theme.ratingbar}>
+        <div className={theme.ratingcomp}>{this.ratingItems}</div>
+        <div className={theme.label}>{total}</div>
```

Comment:
If there is no `total` passed in, we shouldn't be using this line.  Can we make this optional please?